scripts/mkcloud

```diff
@@ -1274,6 +1274,8 @@ Optional
         /root/.ssh/authorized_keys file and to provisioner barclamp
     want_neutronsles12=1 (default 0)
         if there is a SLE12 node, deploy neutron-l3 role into the SLE12 node
+    want_dvr=1 (default='')
```

Comment:
Shouldn't it be `(default 0)`? Like in the other lines above?

Although it seems to be the style you picked in some other lines.